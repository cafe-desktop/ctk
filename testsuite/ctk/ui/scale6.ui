<!-- test invalid scale mark element -->
<interface>
  <object class="CtkScale">
    <marks>
      <foo/>
    </marks>
  </object>
</interface>
