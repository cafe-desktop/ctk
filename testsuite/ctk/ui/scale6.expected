ERROR: ctk-builder-error-quark 1
.:5:13 Unsupported tag for CtkScale: <foo>
